torch/_inductor/codegen/cuda/__init__.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+THIRD_PARTY_PATH = "../../../../third_party/"
+CUTLASS_PY_PATH = "cutlass/tools/library/scripts/"
```

Comment:
I don't think this will work with `pip install torch` since the `third_party` folder is not part of our python package.  Will need to copy the needed stuff into our packaged torch/headers folder.

Reply:
`pip install torch` won't work. Currently only local development works.
I think @atalman has some plans to export cutlass files to torch package correctly (there is also an issue tracked in https://github.com/pytorch/pytorch/issues/106991). 

On the other hand, I just noticed that some torch tests are also based on torch package, which blocks the latter PRs in the stack. Will work with Andrey to resolve this issue.